services/TokenMint.proto

```diff
@@ -32,6 +32,10 @@ Mints tokens to the Token's treasury Account. If no Supply Key is defined, the t
 The operation increases the Total Supply of the Token. The maximum total supply a token can have is 2^63-1.
 The amount provided must be in the lowest denomination possible. Example:
 Token A has 2 decimals. In order to mint 100 tokens, one must provide amount of 10000. In order to mint 100.55 tokens, one must provide amount of 10055.
+If both amount and metadata list get filled, a INVALID_TRANSACTION_BODY response code will be returned.
+If the metadata list contains metadata which is too large, a METADATA_TOO_LONG response code will be returned.
+If neither the amount nor the metadata list get filled, a INVALID_TOKEN_MINT_AMOUNT response code will be returned.
```

Comment:
Optional suggestion: I think this is true only to `FUNGIBLE_COMMON`. For `NON_FUNGIBLE_UNIQUE`, even though an empty metadata list also means `INVALID_TOKEN_MINT_AMOUNT`, a response code of `MISSING_METADATA_LIST` or something like that will make more sense.

Also, I think the explanation here should split into 2 different flows for `FUNGIBLE_COMMON` and `NON_FUNGIBLE_UNIQUE`. For example, an `INVALID_TRANSACTION_BODY` should be returned for a `FUNGIBLE_COMMON` with non-empty metadata list or for a `NON_FUNGIBLE_UNIQUE` with positive `amount`. There are different unhappy scenarios for different types of tokens.

nit: an INVALID_TOKEN_MINT_AMOUNT